docs/migration-from-sck.md

```diff
@@ -335,3 +335,32 @@ To delete the SCK deployment, find the name of the deployment using the `helm ls
   * ```index="Your logs index"```
 * Check the metrics index to see if you are receiving metrics from your Kubernetes cluster
   * ```| mcatalog values(metric_name) WHERE index="Your metrics index"```
+
+## Differences between Splunk Connect for Kubernetes and Splunk OpenTelemetry Collector for Kubernetes
+
+### Read logs location
+
+Splunk Connect for Kubernetes by default read containers logs from `/var/log/containers/*`
+Splunk OpenTelemetry Collector for Kubernetes by default read containers logs from `/var/log/pods/*`
+Change is reflected in `source` filed for extracted logs.
+
+### Default `sourcetype` for containers logs
+
+Splunk Connect for Kubernetes define `sourcetype` for containers as `kube:<container_name>`
```

Comment:
By default, SCK defines `sourcetype` for containers as `kube:container:<container_name>`. It explicitly defines 
`sourcetype` as `kube:<container_name>` for Kubernetes core components(i.e. kubelet, etcd, API server etc).
It is also configurable here: https://github.com/splunk/splunk-connect-for-kubernetes/blob/2cae9b12bbd6545c9ef09b23e619b9783d9ceb38/helm-chart/splunk-connect-for-kubernetes/values.yaml#L359